mirrord/config/src/container.rs

```diff
@@ -51,4 +53,21 @@ pub struct ContainerConfig {
     /// Defaults to `"/opt/mirrord/lib/libmirrord_layer.so"`.
     #[config(default = "/opt/mirrord/lib/libmirrord_layer.so")]
     pub cli_image_lib_path: String,
+
+    /// ### container.override_host_ip {#container-override_host_ip}
+    ///
+    /// Allow the setting of override ip addr for intproxy to use when using mirrord containers
+    /// feature
+    ///
+    /// ```json5
+    /// {
+    ///   "container": {
+    ///     "override_host_ip": "172.17.0.1" // usual resolution of value from `host.docker.internal`
+    ///   }
+    /// }
+    /// ```
+    ///
+    /// This should be useful if your host machine is exposed via different ip addr than one bound
+    /// as host
```

Comment:
```suggestion
    /// ### container.override_host_ip {#container-override_host_ip}
    ///
    /// Allows to override the IP address for the internal proxy to use
    /// when connecting to the host machine from within the container.
    ///
    /// ```json5
    /// {
    ///   "container": {
    ///     "override_host_ip": "172.17.0.1" // usual resolution of value from `host.docker.internal`
    ///   }
    /// }
    /// ```
    ///
    /// This should be useful if your host machine is exposed with a different IP address than the one
    /// bound as host.
```